designs/administrator/future-openshift/clickable-badge/index.md

```diff
@@ -0,0 +1,59 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Clickable Badges
+
+Resource badges can be made clickable to expose further info about the resources.
+By making the badge clickable, we can surface further information pulled from the "Explore" menu item, thus surfacing the "Explore" content on the go.
+This could also reduce confusion around acronyms.
+
+
+
+## General behaviour
+![Clicking on a badge](img/Clickable-Badge-00.png)
+* Currently, resources are implemented to have the name clickable. Badges are currently not clickable and detached from the resource link.    
+* On clicking the resource badge, a popover appears displaying the following:
+  * Full acronym breakdown as the title of the popover.
+  * Resource description, pulled from the Explore>Resource>Details>Description.
+  * Link to the Explore>Resource page.
+
+### Specs
+
+![img](img/Clickable-Badge-02.png)
+* All badges placed in-page or within a modal can be made clickable, including the following placements:
+  * Resource details page title
+  * Within a list.
+  * Within an infobox
+  * Within a modal (in page)
+* Badges that appear on drop-downs should be excluded and not be made clickable.
+* the badge popover should be triggered by clicking, but also by hovering with a 0.5 seconds timeout.
+
+![img](img/Clickable-Badge-01.png)
+* The badge should have a clickable active area with the same height as a primary button
+* This active area should be the same length as the badge (no margins).
+* On hover, the curser should be changed to "clickable object"
+* In the current badge implementation (OS 4.8), no visual change will be made to the badge on hover or click.
+
+
+## Future expansions
```

Comment:
I don't know if these suggestions completely make sense as is because we do already surface this information today in a side panel. We just only show it in the context of the YAML tab, but I think what you are proposing is actually to surface that information beyond just the YAML tab which could be neat. We probably just need a consistent way to access that info from a details page. Do you think just through the badge is enough for discoverability? Brie explored some ideas in the past along these lines: 
![explore-access-1](https://user-images.githubusercontent.com/18057044/119185666-fff94200-ba44-11eb-83e9-b46f66fb00f5.png)
But definitely needs more thought on the integrating with the topology side panel like you and Serena already discussed.